addons/campsite/functions/fn_buildCampsite.sqf

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+/*
+ * Author: Gilles
+ *
+ * TODO DOCSTRING
+ *
+ * Arguments:
+ * 0: xyz
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ *
+ * Public: No
+ *
+ * Scope: Client
+ */
+
+params["_structure"];
```

Comment:
Same es in the other function. What's the point of you having a switchcase for 0, 1 in regards of the term "_structure". Are you trying to do Object Orientated SQF? Or where does this convention come from? Just curious